### PR TITLE
Feature/gh 5545 textfield add to cart

### PR DESCRIPTION
**Pull request by steinsebastian**

### CODE REVIEW

File: projects/core/src/configurator/textfield/facade/configurator-textfield.service.ts [hunk 1]
```diff
@@ -21,4 +33,84 @@ export class ConfiguratorTextfieldService {
...
+            this.createNewConfigurationWithChange(
+              changedAttribute,
+              oldConfiguration
+            )
+          )
+        );
+      });
+  }
+
+  createNewConfigurationWithChange(
```
Comment (schuppell) [inline projects/core/src/configurator/textfield/facade/configurator-textfield.service.ts]: Move down to helper methods

File: projects/core/src/configurator/textfield/connectors/converters.ts [hunk 2]
```diff
@@ -9,3 +9,7 @@ export const CONFIGURATION_TEXTFIELD_NORMALIZER = new InjectionToken<
 export const CONFIGURATION_TEXTFIELD_SERIALIZER = new InjectionToken<
```
Comment (schuppell) [inline projects/core/src/configurator/textfield/connectors/converters.ts]: Where is this converter used?

File: projects/core/src/occ/occ-models/occ-endpoints.model.ts [hunk 3]
```diff
@@ -270,6 +270,12 @@ export interface OccEndpoints {
    * @member {string}
    */
   createConfigurationTextfield?: string;
+  /**
+   * Endpoint for add textfield configuration to cartf
```
Comment (schuppell) [inline projects/core/src/occ/occ-models/occ-endpoints.model.ts]: Typo

File: projects/storefrontlib/src/cms-components/configurator/textfield/textfield-configurator.module.ts [hunk 4]
```diff
@@ -50,12 +51,14 @@ import { ConfigTextfieldInputFieldComponent } from './config-textfield-input-fie
   declarations: [
     ConfigTextfieldFormComponent,
     ConfigTextfieldInputFieldComponent,
+    ConfigTextfieldAddToCartButtonComponent,
   ],
   exports: [ConfigTextfieldFormComponent, ConfigTextfieldInputFieldComponent],
```
Comment (schuppell) [inline projects/storefrontlib/src/cms-components/configurator/textfield/textfield-configurator.module.ts]: Missing component. ConfigTextfieldAddToCartButtonComponent should be exported as well

File: projects/core/src/configurator/textfield/facade/configurator-textfield.service.ts (hunk 1 above)
Comment (steinsebastian) [reply projects/core/src/configurator/textfield/facade/configurator-textfield.service.ts]: done

File: projects/core/src/configurator/textfield/connectors/converters.ts (hunk 2 above)
Comment (steinsebastian) [reply projects/core/src/configurator/textfield/connectors/converters.ts]: nowehere

File: projects/core/src/occ/occ-models/occ-endpoints.model.ts (hunk 3 above)
Comment (steinsebastian) [reply projects/core/src/occ/occ-models/occ-endpoints.model.ts]: done

File: projects/storefrontlib/src/cms-components/configurator/textfield/textfield-configurator.module.ts (hunk 4 above)
Comment (steinsebastian) [reply projects/storefrontlib/src/cms-components/configurator/textfield/textfield-configurator.module.ts]: done

**Pull request merged**